lib/PuppeteerSharp.Tests/SingleFileDeployment/PublishedSelfContainedBinaryShouldWork.cs

```diff
@@ -0,0 +1,94 @@
+using System.Diagnostics;
+using System.IO;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace PuppeteerSharp.Tests.SingleFileDeployment
+{
+    public class PublishedSelfContainedBinary
```

Comment:
Let's name this file `PublishedSelfContainedBinary.cs`

Reply:
SelfContained != SingleFile 

> Publishing your app as self-contained produces an application that includes the .NET runtime and libraries, and your application and its dependencies. Users of the application can run it on a machine that doesn't have the .NET runtime installed.

SelfContained refers to the `runtime` being included with the application. 

> Single-file deployment is available for both the [framework-dependent deployment model](https://learn.microsoft.com/en-us/dotnet/core/deploying/#publish-framework-dependent) and [self-contained applications](https://learn.microsoft.com/en-us/dotnet/core/deploying/#publish-self-contained).

For reference.

If the intention is to test `Single File` build then one of the following is probably more descript:

- `PublishSingleFileTests`
- `PublishSingleFileDeploymentTests`

Reply:
I choosed 'PublishSingleFileTests'